gef-ui/webpack.common.config.js

```diff
@@ -23,6 +23,12 @@ module.exports = {
       new CssMinimizerPlugin({}),
     ],
   },
+  resolve: {
+    alias: {
+      'node_modules/govuk-frontend': path.resolve(__dirname, '../node_modules/govuk-frontend'),
+      'node_modules/@ministryofjustice': path.resolve(__dirname, '../node_modules/@ministryofjustice'),
+    },
+  },
```

Comment:
This change is also needed because of the hoisting of dependencies to the root `node_modules`. We can change SCSS files we control from e.g.
```
@import "node_modules/govuk-frontend/govuk/all";
```
to
```
@import "../node_modules/govuk-frontend/govuk/all";
```
but these imports refer to other imports that are part of the package, so not in our control to change. This alias makes the change to all relevant files when building the CSS output.